File: cppTopy/programs1/binary.cpp


#include "binary.h"
#include <cmath>

int binary_to_int(std::vector<int> binary) {
    int sum=0;
    int count=0;
    for(int i : binary) {
        
        sum+=std::pow(2,count)*i;
        count=count+1;
    }
    return sum;
}

File: cppTopy/programs1/binary.py
# -*- coding: utf-8 -*-
"""
Created on Sun Jan 23 12:19:36 2022

@author: 123
"""

def binary_to_int(binary):
    m_sum = 0
    count = 0
    
    for i in binary:
        
        m_sum += (2**count) * i
        count += 1
        
    return m_sum